tests/e2e/conftest.py

```diff
@@ -78,14 +82,29 @@ def init_project(
     protostar_bin: Path,
     project_name: str,
 ) -> ProjectInitializer:
-    def _init_project(override_project_name: Optional[str] = None) -> None:
+    def _init_project(
+        override_project_name: Optional[str] = None, cairo1: Optional[bool] = False
+    ) -> None:
         if override_project_name is None:
             real_project_name = project_name
         else:
             real_project_name = override_project_name
 
         child = pexpect.spawn(f"{protostar_bin} init {real_project_name}")
         child.expect(pexpect.EOF, timeout=30)
+        if cairo1:
+            project_dir = Path(os.getcwd()) / "foobar"
+            main_path = project_dir / "src" / "main.cairo"
+            fixture_path = (
+                Path(__file__).parent.parent.parent
+                / "tests"
+                / "e2e"
+                / "fixtures"
+                / "basic_cairo1.cairo"
+            )
+            assert main_path.exists() and fixture_path.exists()
+            main_path.write_text(fixture_path.read_text())
+            Path(project_dir / "build").mkdir()
```

Comment:
Once `init-cairo1` is done in https://github.com/software-mansion/protostar/issues/1496, we can just probably do
```
child = pexpect.spawn(f"{protostar_bin} init-cairo1 {real_project_name}")
```